src/test/java/unstable/KeyboardShortcutsConfigTest.java

```diff
@@ -105,14 +108,43 @@ public void repeatedKeySpecified() {
     }
 
     private static void reloadPrefs(Preferences prefs) {
-        prefs.saveGlobalConfig();
-        String loadGlobalConfig = "loadGlobalConfig";
+        String loadConfigMethodString = "loadConfig";
+        String sessionConfigFieldString = "sessionConfig";
+        String userConfigFieldString = "userConfig";
         try {
-            Method method = Preferences.class.getDeclaredMethod(loadGlobalConfig);
-            method.setAccessible(true);
-            method.invoke(prefs);
-        } catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e) {
-            fail(String.format("Problem invoking private method; does %s still exist?", loadGlobalConfig));
+            Method loadConfigMethod =
+                    Preferences.class.getDeclaredMethod(loadConfigMethodString,
+                            String.class,
+                            String.class,
+                            Class.class);
+            Field sessionConfigField = Preferences.class.getDeclaredField(sessionConfigFieldString);
+            Field userConfigField = Preferences.class.getDeclaredField(userConfigFieldString);
+
+            loadConfigMethod.setAccessible(true);
+            sessionConfigField.setAccessible(true);
+            userConfigField.setAccessible(true);
+            sessionConfigField.set(prefs,
+                    loadConfigMethod.invoke(prefs,
+                            TestController.TEST_DIRECTORY,
+                            TestController.TEST_SESSION_CONFIG_FILENAME,
+                            SessionConfig.class));
+            userConfigField.set(prefs,
+                    loadConfigMethod.invoke(prefs,
+                            TestController.TEST_DIRECTORY,
+                            TestController.TEST_USER_CONFIG_FILENAME,
+                            UserConfig.class));
+
+        } catch (NoSuchMethodException e) {
+            fail(String.format("Problem invoking private method; does %s still exist? %s",
+                    loadConfigMethodString,
+                    e.toString()));
+        } catch (NoSuchFieldException e) {
+            fail(String.format("Problem accessing private field; do %s and %s still exist? %s",
+                    sessionConfigFieldString,
+                    userConfigFieldString,
+                    e.toString()));
+        } catch (IllegalAccessException | InvocationTargetException e){
```

Comment:
Not sure if it is worth to have all these catch blocks. Just `throw exception` instead? If a test fails, the stack trace should tell us where the problem is.